sessions/js-structure/quiz-app/global.css

```diff
@@ -0,0 +1,59 @@
+:root {
+	--primary: #ffffff;
+	--secondary: #ff4a11;
+	--secondary-light: #fddcd2;
+	--water: #f3f5f9;
+	--black: #252629;
+	--ff-primary: sans-serif;
```

Comment:
```suggestion
	--black: #252629;
	
	--ff-primary: sans-serif;
```

Reply:
I usually prefer having some 'space' between different types of variables, but it's ultimately not dramatically important.